src/context_store/storage/sqlite.py

```
@@ -758,7 +758,23 @@ async def vector_search(
     async def keyword_search(
         self, query: str, top_k: int, project: str | None = None
     ) -> list[ScoredMemory]:
-        """Full-text search using FTS5."""
+        """Full-text search using FTS5.
+
+        FTS5 サニタイズ方針:
+        - クエリをホワイトスペースでトークン分割し、各トークンを個別にダブルクォートで囲む
+        - これにより FTS5 特殊文字 (``*``, ``AND``, ``OR``, ``NOT``, ``NEAR``, ``?``) が
+          トークン内でエスケープされつつ、トークン間は暗黙 AND として扱われる
+        - マルチワードクエリ ``"machine learning"`` は各ワードがドキュメント内の任意の位置に
+          存在すればマッチする（フレーズ一致ではない）
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)